client/src/Components/ToggleStandardTime.js

```diff
@@ -41,10 +41,16 @@ class ToggleStandardTime extends React.Component {
         exclusive
         onChange={this.handlePerformanceTypeChange}
       >
-        <TooltipToggleButton value="celerityTime" title="Dev time compared to Ticket points">
+        <TooltipToggleButton
+          value="celerityTime"
+          title="Ticket is KO when dev time is over ticket points"
```

Comment:
Ticket is KO when dev time is greater than the time computed from the number of ticket points ?